src/features/creators/newScriptProject.ts

```diff
@@ -0,0 +1,18 @@
+import { MarkdownString, window } from 'vscode';
+import { PythonProject, PythonProjectCreator, PythonProjectCreatorOptions } from '../../api';
+import { PythonProjectManager } from '../../internal.api';
+
+export class NewScriptProject implements PythonProjectCreator {
+    public readonly name = 'newScript';
+    public readonly displayName = 'Script';
+    public readonly description = 'Create a new Python script project';
+    public readonly tooltip = new MarkdownString('Create a new Python script with basic structure');
+
+    constructor(private readonly pm: PythonProjectManager) {}
+
+    async create(_options?: PythonProjectCreatorOptions): Promise<PythonProject | undefined> {
+        // show notification that the script creation was selected than return undefined
+        window.showInformationMessage('Creating a new Python script project...');
```

Comment:
Python script project feels a little wordy, I wouldn't want to just say script since they will end up with more than a single file so again maybe opting to just say "Python project"
```suggestion
        window.showInformationMessage('Creating a new Python script project...');
```